apis/cloudstack/src/test/java/org/jclouds/cloudstack/features/VirtualMachineApiTest.java

```diff
@@ -204,11 +205,14 @@ public void testChangeServiceForVirtualMachine() throws SecurityException, NoSuc
    }
 
    public void testUpdateVirtualMachine() throws SecurityException, NoSuchMethodException, IOException {
-      Invokable<?, ?> method = method(VirtualMachineApi.class, "updateVirtualMachine", String.class);
-      GeneratedHttpRequest httpRequest = processor.createRequest(method, ImmutableList.<Object> of(5));
+      Invokable<?, ?> method = method(VirtualMachineApi.class, "updateVirtualMachine", String.class,
+              UpdateVirtualMachineOptions.class);
+      GeneratedHttpRequest httpRequest = processor.createRequest(method, ImmutableList.<Object> of("5",
+              UpdateVirtualMachineOptions.Builder.group("test")));
```

Comment:
Sure, can do that. Saw some other tests also tested just one property, but I'll adjust it.